pyproject.toml

```diff
@@ -181,7 +182,7 @@ tdengine = [
     "taos-ws-py>=0.3.8"
 ]
 teradata = ["teradatasql>=16.20.0.23"]
-thumbnails = ["Pillow>=10.0.1, <11"]
+thumbnails = ["Pillow>=10.0.1, <12"]
```

Comment:
If `Pillow` is already a hard dependency as defined above, do we really need this option any more?